api/users.go

```diff
@@ -55,6 +59,12 @@ func PatchUser(c *gin.Context) {
 		c.AbortWithStatus(http.StatusNotFound)
 		return
 	}
+
+	if session.User(c).ID == user.ID {
+		c.String(422, "Users can't deactivate or change admin rights themselfs")
+		return
```

Comment:
with 0.5 user administrators are declared via environment variable so this no longer applies